test/parse_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule ParseTest do
+  use ExUnit.Case
+
+  @urls [
+    "http://onigumo.local/hello.html",
+    "http://onigumo.local/bye.html"
+  ]
+
+  test("Parsing values of href attributes in html links") do
+    document = ~s("<!doctype html>
```

Comment:
Does the `~s` sigil trim the whitespace from the beginning on the following lines? Does the string contain `<!doctype html>\n<html>…` or `<!doctype html>\n      <html>…`? The former is correct; the latter is not.

Reply:
The [s sigil](https://hexdocs.pm/elixir/1.14/Kernel.html#sigil_s/2) does not trim whitespace from the beginning of the line. Although the indentation looks nice in the code, it would incorrectly propagate to the inserted HTML. HTML is insensitive to whitespace, so the indentation does not cause any harm, but it still is an unwanted interference.